chapters/introduction/lab/content/setup/setup.md

```diff
@@ -0,0 +1,141 @@
+# Setting up your environment
+
+In this section we will install all the requirements needed for the practical sessions.
+This includes the following:
+- rustup - The Rust toolchain installer
+- MetaMask - Ethereum wallet
+- mxpy - tool for interacting with the blockchain
+- sc-meta - tool for compiling smart contracts
```

Comment:
sc-meta is not used only for compiling SC: better use "universal smart contract management tool"